tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "noImplicitAny": false,
```

Comment:
I'm not sure we want implicit any - do you have a specific reason for this?

Reply:
So this is my lack of experience with typescript. It was a quick solution to a ts error that pops up without it. In the `Editor.tsx` component, for example, it tells me:
```
Binding element 'feed' implicitly has an 'any' type.
```

Reply:
Fixed the typescript with this